pandas/compat/chainmap.py

```diff
@@ -6,12 +6,18 @@ def __setitem__(self, key, value):
         for mapping in self.maps:
             if key in mapping:
                 mapping[key] = value
-                return
+                break
```

Comment:
does break vs return make a difference?  here or below?

Reply:
> does break vs return make a difference? here or below?

No it doesn't (not here at least), But I think it's making the code more readable.
I don't mind reverting those changes if you think they're doing the opposite.

Reply:
Doesn't this always now run line 10 whereas before it wouldn't previously if the function returned here?

Reply:
\*pacepalm\* Yes you're 100% right.